php/php.editor/src/org/netbeans/modules/php/editor/csl/SemanticAnalysis.java

```diff
@@ -727,11 +742,17 @@ public void visit(ConstantDeclaration node) {
                 parentNode = path.get(1);
             }
             if (parentNode instanceof ClassDeclaration || parentNode instanceof InterfaceDeclaration
-                    || parentNode instanceof TraitDeclaration) {
+                    || parentNode instanceof TraitDeclaration || parentNode instanceof ClassInstanceCreation) {
```

Comment:
Maybe, we can delete trait checking because trait can't declare constants. But I suppose it is harmless even as it is.

Reply:
You are right - removed.